### PR TITLE
Add fpm handling in docker-entrypoint.sh

### PR DESCRIPTION
**Pull request by landbit**
Hi
Feel free to merge this small modification to also handle fpm in entrypoint.sh.
The script had a test that was executed only for apache variant.
cheers

### CODE REVIEW

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-if [[ "$1" == apache2* ]]; then
+if [[ "$1" == apache2* ]] || [[ "$1" == php-fpm ]]; then
```
Comment (md5) [inline apache/docker-entrypoint.sh]: The new test can be a simple `[ "$1" == php-fpm ]` since it doesn't have a wildcard in it.

**Comment by md5** [comment]
Good catch! :+

**Comment by landbit** [comment]
md5, I've pushed a fix regarding your last comment, cheers.

**Comment by leon** [comment]
It would be awesome if we could merge this, the fpm support is unusable at the moment

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**